bin/update-readmes.js

```diff
@@ -14,6 +40,7 @@ packages.forEach( ( packageName ) => {
 		`packages/${ packageName }/src/index.js`,
 		`--output packages/${ packageName }/README.md`,
 		'--to-token',
+		'--ignore "unstable|experimental"',
```

Comment:
Couple thoughts:

- If this is a regular expression, should we be assuring `^` so that we'd not inadvertently ignore the token `definitely_not_unstable` (ignoring the fact this would, for other reasons, not be allowed)
- The [guidelines](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-apis) clearly prescribe the double-underscore `__` prefix. I don't think we want the tolerance here. I want to see clear and loudly in the produced documentation if we're not naming these correctly.

Reply:
Yeah, I wanted a bit of leeway on purpose. There are a couple of offenders right now on `master`. Created https://github.com/WordPress/gutenberg/issues/14337 to review this after all packages are set up.